eos/constraint.cc

```diff
@@ -162,6 +175,38 @@ namespace eos
 
             return os;
         }
+
+        virtual void serialize(YAML::Emitter & out) const
```

Comment:
If this is, what should we do with the arcane `as_string` methods in `LogLikelihood`? Do you want to update them, too? Or just keep it as it is because it's working?

Reply:
```as_string``` is used in the existing data dumps to HDF. So better keep it for now.

Reply:
Ok